website/static/js/fangorn.js

```diff
@@ -2441,9 +2439,7 @@ function isInvalidDropItem(folder, item, cannotBeFolder, mustBeIntra) {
         // no dropping if waiting on waterbutler ajax
         item.inProgress ||
         (cannotBeFolder && item.data.kind === 'folder') ||
```

Comment:
Folders can be dragged into the figshare root if the root is a Project.  Otherwise, it should be disallowed.  Code and tests updated to reflect this.